docker/hvd/build_all.sh

```diff
@@ -30,7 +30,14 @@ cd $curr_dir/hvd
 set -eu
 
 image_tag=""
-pth_version=${PTH_VERSION:-1.6.0-cuda10.1-cudnn7}
+
+if [[ -z "${PTH_VERSION}" ]]; then
+    echo "PTH_VERSION is not set"
+    exit 1
+else
```

Comment:
We can define `pth_version=${PTH_VERSION}` without else for better understanding.

Reply:
yes of course